src/identities/CommandClaim.ts

```diff
@@ -12,19 +13,14 @@ class CommandClaim extends CommandPolykey {
     this.name('claim');
     this.description('Claim a Digital Identity for this Keynode');
     this.argument(
-      '<providerId>',
-      'Name of the digital identity provider',
-      binParsers.parseProviderId,
-    );
-    this.argument(
-      '<identityId>',
-      'Digital identity to claim',
-      binParsers.parseIdentityId,
+      '<gestaltId>',
+      '`Provider ID:Identity ID`',
+      binParsers.parseGestaltId,
```

Comment:
This command is for claiming an identity right? We Only want a IdentityId to be valid. `parseGestaltId` can parse `NodeId`s as well, so  we don't want that here. 

I'm pretty sure with this the command will take a `NodeId` but break.